### PR TITLE
Add FPGA accelerated examples

### PR DESCRIPTION
**Pull request by eliaskoromilas**
**What this PR does / why we need it**:

* Provides instructions on how to accelerate Katib Experiments on AWS F1 instances.
* Includes an accelerated XGBoost Parameter Tuning example.

### CODE REVIEW

**Comment by egkiastas** [comment]
@googlebot I consent.

File: examples/v1beta1/fpga/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```sh
+kubectl get nodes -o custom-columns=NAME:metadata.name,FPGAS:.status.capacity.xilinx/aws-vu9p-f1
+```
+
+## Experiment
+
+You can submit a new accelerated Experiment and check your Experiment results
+using the Web UI, as usual.
+
+#### XGBoost Parameter Tuning [[source](https://github.com/inaccel/jupyter/blob/master/lab/dot/XGBoost/parameter-tuning.py)]
```
Comment (andreyvelich) [inline examples/v1beta1/fpga/README.md:66]: Please, can you also point [here](https://github.com/kubeflow/katib/tree/master/examples/v1beta1#list-of-current-katib-training-container-images) that source code for the `docker.io/inaccel/jupyter:lab` image users can find in that link?

**Comment by andreyvelich** [comment]
I believe you need to rebase to pass the tests.

**Comment by eliaskoromilas** [comment]
> I believe you need to rebase to pass the tests.

the branch is now only ahead of upstream/master

**Pull request merged**